recipes/pypsa/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "pypsa" %}
+{% set version = "0.14.1" %}
+{% set sha256 = "477716d83806a29e75bcb48c3e7c19b8f96ca1af68d7eaf6fd86202f3c79c913" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```

This can be simplified now:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv "
```